test_runtime.sh

```diff
@@ -77,9 +77,9 @@ trap cleanup EXIT
 tar -xf  rootfs.tar.gz -C ${TESTDIR}
 cp runtimetest ${TESTDIR}
 
-ocitools generate --output "${TESTDIR}/config.json" "${TEST_ARGS[@]}" --rootfs '.'
+oci-runtime-tool generate --output "${TESTDIR}/config.json" "${TEST_ARGS[@]}" --rootfs '.'
 
-TESTCMD="${RUNTIME} start $(uuidgen)"
+TESTCMD="${RUNTIME} run $(uuidgen)"
```

Comment:
Drive-by change?  More discussion on this change in #184, but I'd rather not do this as part of this PR.